### PR TITLE
Fixing time zone and testing cases

### PR DESCRIPTION
**Pull request by verdotte**
#### What does this PR do?
fixing the time zone problem and solving follow test cases

#### Description of Task to be completed?
-implement a method for returning the current date & time according to the default time zone which is Africa/Kinshasa
-apply that method in all the models which capture the current date & time

#### How should this be manually tested?
run npm test follow.test.js since only that file was involving in the test

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by gracelungu — APPROVED** [review]

**Comment by gracelungu** [comment]
@verdotte Good work. But try to improve your PR description.

**Comment by verdotte** [comment]
[Fix the issue with the date and update the TimeAgo component](https://trello.com/c/boaFJD2I/53-fix-the-issue-with-the-date-and-update-the-timeago-component)

File: src/helpers/defaultDateTime.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import moment from 'moment-timezone';
+
+const defaultDateTime = () => moment
+  .tz(Date.now(), 'Africa/Kinshasa')
+  .format();
+
+export default defaultDateTime;
```
Comment (oesukam) [inline src/helpers/defaultDateTime.js]: New line required at the end of each file

**Review by oesukam — CHANGES_REQUESTED** [review]

**Review by oesukam — APPROVED** [review]
LGTM

**Pull request merged**